src/components/Message/WarningToken.vue

```diff
@@ -1,17 +1,18 @@
 <template>
   <div id="error" class="d-flex flex-items-center px-4 py-2">
```

Comment:
I would just add a bit more space here otherwise it looks great!
```suggestion
  <div id="error" class="d-flex flex-items-center px-4 py-3">
```
Before
![image](https://user-images.githubusercontent.com/16245250/103763942-2c3f9380-504d-11eb-92d8-1b8c34142bee.png)

After
![image](https://user-images.githubusercontent.com/16245250/103763923-1d58e100-504d-11eb-8dd9-3b5e7a8797ef.png)

Reply:
Good one 👍